### PR TITLE
Add embedded ansible post three

### PR DESCRIPTION
**Pull request by d-m-u**
Peter wrote 4 articles about embedded Ansible work in CFME 4.5 that were intended for posting on the CloudFormsNow site but aren't going to all get posted and so I was asked to put them here since it's a shame to waste the research and time that went into this effort. They're great, and we should post them, part three.

Part one: https://github.com/ManageIQ/manageiq.org/pull/796
Part two: https://github.com/ManageIQ/manageiq.org/pull/797

### CODE REVIEW

**Comment by d-m-u** [comment]
@cybette and @pemcg could I please have your 👀 on this as well? thanks a much

File: site/_posts/2020-02-13-Embedded-Ansible-Part-3-Ansible-In-An-Automation-Workflow.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+TASK [Get the "ldap_user_password" encrypted attribute from a configuration domain instance (get_decrypted_attribute)] ***
+ok: [localhost]
+
+TASK [debug] *******************************************************************
+ok: [localhost] => {
+    "msg": "Result:{u'attribute': u'ldap_user_password', u'object': u'/Configuration/OpenShift/Parameters/default', u'value': u'secret_ldap_password'}"
+}
+```
+
+Attributes can also be written back into workspace objects (usually the root object) using the `set_attribute`, `set_attributes` and `set_encrypted_attriibute` functions, as follows. Attributes written in this way are readable by a subsequent Ruby method in the workflow as `$evm.root[‘<attribute_name>’]`.
```
Comment (cybette) [inline site/_posts/2020-02-13-Embedded-Ansible-Part-3-Ansible-In-An-Automation-Workflow.md]: `set_encrypted_attriibute` - is that a typo or is the function name actually spelled with an extra `i`?

Comment (d-m-u) [reply site/_posts/2020-02-13-Embedded-Ansible-Part-3-Ansible-In-An-Automation-Workflow.md]: Yarps thanks

**Comment by pemcg** [comment]
Possibly some minor formatting typos. On line 64 it might be cleaner to line-wrap the 'u'/pemcg/stuff/...'. Also where you describe the set_attribute function (line 204) the two related functions set_attributes and set_encrypted_attribute should be their own sub-headings but appear to be wrapped into the code section. Other than that it looks good.

**Comment by pemcg** [comment]
LGTM 👍

**Comment by cybette** [comment]
Link to part 1: https://www.manageiq.org/blog/2020/02/Embedded-Ansible-Part-1-Built-In-Roles/
Link to part 2: https://www.manageiq.org/blog/2020/02/Embedded-Ansible-Part-2-Passing-Parameters-Into-A-Playbook/

File: site/_posts/2020-02-13-Embedded-Ansible-Part-3-Ansible-In-An-Automation-Workflow.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+|    |    /ManageIQ/Infrastructure/VM/Provisioning/StateMachines/VMProvision_vm/template  ($evm.parent)
+|    |    |    /ManageIQ/Infrastructure/VM/Provisioning/StateMachines/Methods/CustomizeRequest
+|    |    |    /Configuration/OpenShift/Parameters/default
+|    |    |    /Investigative_Debugging/Discovery/ObjectWalker/object_walker  ($evm.object)
+|    |    |    |    /Investigative_Debugging/Discovery/ObjectWalker/configuration
+```
+
+The stored attributes of any object in the workspace can be retrieved from a running Ansible playbook method, which can also save attributes and values back to the root object.
+
+The workspace can be accessed from a running playbook method using the *manageiq-core.manageiq-automate* role described in part one of this series.
```
Comment (cybette) [inline site/_posts/2020-02-13-Embedded-Ansible-Part-3-Ansible-In-An-Automation-Workflow.md]: wanna add link to part one here?

**Pull request merged**